tests/unit/test_utils.py

```diff
@@ -2,36 +2,21 @@
 import logging
 import unittest.mock as mock
 
-from jsonschema import ValidationError
-
-from broadway_api.utils.bootstrap import (
-    initialize_cluster_token,
-    initialize_course_tokens,
-)
+from broadway_api.utils.bootstrap import initialize_course_tokens
 from broadway_api.daos.course import CourseDao
 from tests.base import BaseTest
 
 logging.disable(logging.WARNING)
 
 
-class TestClusterTokenUtils(BaseTest):
-    def test_cluster_token_from_env(self):
```

Comment:
We can still test this right? We can extract the token from `self.app.settings`?